src/import/sst_service.rs

```diff
@@ -185,6 +185,17 @@ where
                 .ingest_maybe_slowdown_writes(CF_WRITE)
                 .expect("cf")
         {
+            match self.engine.get_sst_key_ranges(CF_WRITE, 0) {
+                Ok(l0_sst_ranges) => {
+                    warn!(
+                        "sst ingest is too slow";
+                        "sst_ranges" => ?l0_sst_ranges,
```

Comment:
Do you want to use log_wrapper to convert it to hex and add auto-redaction support?

Reply:
checked that in order to know the problem we can't use redact 😂